contents/docs/data/identify.mdx

```diff
@@ -0,0 +1,466 @@
+    },
+})
+```
+
+```javascript
+posthog.identify(
+    'distinct_id',
+    { email: 'max@hedgehogmail.com', name: 'Max Hedgehog' } 
+    {}, 
+);
+```
+
+```python
+posthog.identify(
+    'distinct_id',
+    {
+        'name': 'Max Hedgehog',
+        'email': 'max@hedgehogmail.com'
+    }
+)
+```
+
+```php
+PostHog::identify(array(
+  'distinctId' => 'distinct_id',
+  'properties' => array(
+    'name' => 'Max Hedgehog'
+    'email' => 'max@hedgehogmail.com',
+  )
+));
+```
+
+```ruby
+posthog.identify({
+  distinct_id: "distinct_id",
+  properties: {
+    name: "Max Hedgehog",
+    email: "max@hedgehogmail.com"
+  }
+})
+```
+
+```go
+client.Enqueue(posthog.Identify{
+  DistinctId: "distinct_id",
+  Properties: posthog.NewProperties().
+    Set("name", "Max Hedgehog").
+    Set("email", "max@hedgehogmail.com"),
+})
+```
+
+```java
+posthog.identify(
+  "distinct_id",
+  new HashMap<String, Object>() {
+    {
+      put("name", "Max Hedgehog");
+      put("email", "max@hedgehogmail.com");
+    }
+  }
+);
+```
+
+```bash
+curl -v -L --header "Content-Type: application/json" -d '{
+    "api_key": "<ph_project_api_key>",
+    "timestamp": "2020-08-16 09:03:11.913767",
+    "context": {},
+    "distinct_id": "distinct_id",
+    "$set": {},
+    "event": "$identify"
+}' https://app.posthog.com/capture/
+```
+
+</MultiLanguage>
+
+> We don't recommend using emails as distinct IDs, as people changing their email will lead to extra complexity to maintaining data integrity. It is better to use database IDs or account IDs.
+
+On your frontend, you should call identify as soon as you're able to. Typically, this is every time your app loads for the first time, or directly after your users log in. This ensures that events sent during your users' sessions are correctly associated with them. 
+
+### Setting user properties
+
+You'll notice that one of the parameters in the `identify` method is a `properties` object. This enables you to set [user properties](/docs/getting-started/user-properties), which can be used in PostHog features such as [filters](/docs/product-analytics/trends#filtering-events-based-on-properties) or [cohorts](/docs/data/cohorts). 
+
+> Whenever possible, we recommend passing in all user properties you have available each time you call identify, as this will ensure that their user profile on PostHog is up to date.
+
+User properties can also be set using a `capture` and not only with `identify`. See our [user properties docs](/docs/getting-started/user-properties) for more details on how to work with them and best practices.
+
+## Alias
+
+Sometimes, you may want to assign multiple distinct IDs to a single user. This is helpful in scenarios where your primary distinct ID may be inaccessible. For example, if a distinct ID which is typically used on the frontend is not available in certain parts of your backend code. In this case, you can use `alias` to assign another distinct ID to the same user:
+
```

Comment:
While sub-system is technically more correct, since this is just an example with the purpose of illustrating a scenario, I think it's simpler and easier to understand if we use backend or frontend